### PR TITLE
Edit IGetDepartmentsRequest

### PR DESCRIPTION
**Pull request by VLiamin**

### CODE REVIEW

File: src/DepartmentService.Mappers/Models/DepartmentDataMapper.cs [hunk 1]
```diff
@@ -1,25 +1,29 @@
...
         : new DepartmentData(
           dbDepartment.Id,
           dbDepartment.Name,
-          directorUserId: dbDepartment.Users
-            ?.FirstOrDefault(du => du.Role == (int)DepartmentUserRole.Manager)
-            ?.UserId,
+          dbDepartment.ShortName,
           projectsIds: dbDepartment.Projects?.Select(dp => dp.ProjectId).ToList(),
-          usersIds: dbDepartment.Users?.Select(u => u.UserId).ToList());
+          users: dbDepartment.Users.Select(_departmentUserDataMapper.Map).ToList());
```
Comment (MariyaSin) [inline src/DepartmentService.Mappers/Models/DepartmentDataMapper.cs:26]: dbDepartment.Users?.Select(_departmentUserDataMapper.Map).ToList());

File: src/DepartmentService.Mappers/Models/DepartmentUserDataMapper.cs [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  public class DepartmentUserDataMapper : IDepartmentUserDataMapper
+  {
+    public DepartmentUserData Map(DbDepartmentUser dbDepartment)
+    {
+      if (dbDepartment is null)
+      {
+        return null;
+      }
+
+      return new DepartmentUserData(dbDepartment.UserId, (DepartmentUserRole)dbDepartment.Role);
```
Comment (MariyaSin) [inline src/DepartmentService.Mappers/Models/DepartmentUserDataMapper.cs:17]: use ?: operator and remove 12-15 lines

**Review by MariyaSin — APPROVED** [review]

File: src/DepartmentService.Mappers/Models/DepartmentUserDataMapper.cs (hunk 2 above)
Comment (VLiamin) [reply src/DepartmentService.Mappers/Models/DepartmentUserDataMapper.cs]: done

File: src/DepartmentService.Mappers/Models/DepartmentDataMapper.cs (hunk 1 above)
Comment (VLiamin) [reply src/DepartmentService.Mappers/Models/DepartmentDataMapper.cs]: done

**Review by ndoriomedov — APPROVED** [review]

**Pull request merged**